apps/loan/src/components/PageLlamaMarkets/cells/PoolTitleCell.tsx

```diff
@@ -1,20 +1,27 @@
-import { LendingVaultFromApi } from '@/entities/vaults'
+import { LendingVault } from '@/entities/vaults'
 import Box from '@mui/material/Box'
 import TokenIcons from 'main/src/components/TokenIcons'
-import React from 'react'
+import React, { useMemo } from 'react'
 import Typography from '@mui/material/Typography'
+import { CellContext } from '@tanstack/react-table'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
 
-export const PoolTitleCell = ({ data }: { data: LendingVaultFromApi }) => {
-  const coins = Object.values(data.assets)
+const { Spacing } = SizesAndSpaces
+
+export const PoolTitleCell = ({ getValue, row }: CellContext<LendingVault, LendingVault['assets']>) => {
+  const coins = useMemo(() => Object.values(getValue()), [getValue])
+  const { blockchainId } = row.original
+  const urlPath = `curve-assets/images/assets${blockchainId == 'ethereum' ? '' : `-${blockchainId}`}/`
   return (
     <Box display="flex">
       <TokenIcons
-        // todo: is imageBaseUrl dependent on the chain? It should come from the API then.
-        imageBaseUrl="https://cdn.jsdelivr.net/gh/curvefi/curve-assets/images/assets/"
+        imageBaseUrl={`https://cdn.jsdelivr.net/gh/curvefi/${urlPath}`}
         tokens={coins.map((c) => c.symbol)}
         tokenAddresses={coins.map((c) => c.address)}
       />
-      <Typography variant="tableCellL">{coins.map((coin) => coin.symbol).join(' - ')}</Typography>
+      <Typography variant="tableCellL" sx={{ paddingInlineStart: Spacing.sm }}>
```

Comment:
Should we consider the extra padding to be part of the tableCellL variant for Typography, so that it becomes part of the default styling?

Reply:
I don't think so, this is just to separate the text from the icons. I'll change to use flex gap instead

Reply:
Also will move the typography to the column meta